packages/language-server/src/server.ts

```diff
@@ -132,6 +135,11 @@ connection.onNotification(
   },
 );
 
+connection.onNotification('updateParameters', (parameters: Neo4jParameters) => {
+  neo4jSchemaPoller.setParameters(parameters);
+  relintAllDocuments();
+});
+
```

Comment:
When we set the parameters in the UI layer, we want them to be available for the language server to complete